code/controllers/subsystems/jobs.dm

```diff
@@ -10,6 +10,11 @@ var/const/SUP               =(1<<8)
 var/const/SPT               =(1<<9)
 var/const/EXP               =(1<<10)
 var/const/ROB               =(1<<11)
+var/const/LCZ               =(1<<12)
+var/const/ECZ               =(1<<13)
+var/const/HCZ               =(1<<14)
+var/const/BUR				=(1<<15)
+var/const/REP				=(1<<16)
```

Comment:
replace all of this with defines.........

Reply:
That would mean throwing out the already existing department flags system (and everything that relies on it), unless we can do flags with defines somehow?